Detections/ASimNetworkSessions/SeveralDenyActionsRegistered.yaml

```diff
@@ -0,0 +1,40 @@
+id: D66DD819-F2E6-43F7-8BC1-70BDE8011B39
+name: Several deny actions registered (AsimWebSession)
```

Comment:
Let's try this time to support [dynamic name and description](https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#alertdetailsoverride).

For example
alertDisplayNameFormat: Several deny actions registered for source {{SrcIpAddr}} accessing {{DstIpAddr}} using {{NetworkApplicationProtocol}} protocol.
 - Should be URL for the proxy version.
 - Need to make sure NetworkApplicationProtocol has a value. Can be set to "unknown" if no value.

alertDescriptionFormat: Potential attacker at address  may have tried to move, or scan, from {{SrcIpAddr}} to {{DstIpAddr}} on the network using {{NetworkApplicationProtocol}} protocol and and was denied more than 5 times.